src/Exercism.Representers.CSharp/Normalization/SyntaxNodeSimplifier.cs

```diff
@@ -23,7 +24,10 @@ internal static class SyntaxNodeSimplifier
             new RemoveUsingDirectives(),
             new RemoveComments(),
             new NormalizeIdentifiers(mapping),
-            new NormalizeWhiteSpace()
+            new NormalizeDictionaryInitialization(),
+            // NormalizeWhiteSpace() must be last rewriter as
+            // the unit test expected values have a whitespace dependency
+            new NormalizeWhiteSpace(),
```

Comment:
I'm wondering if instead of having this comment, we should not have this be subject to a possible re-arrangement by any contributors by doing it slightly different. I think we might be better off removing the `NormalizeWhiteSpace` rewriter altogether and calling `.NormalizeWhitespace()` at the end of the `SyntaxRewriters(mapping).Aggregate(node, (acc, rewriter) => rewriter.Visit(acc));` call above.

We can do this in a separate PR.

Reply:
Done

Reply:
And by done you mean: will be done in a separate PR?